### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by xyz1008**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "startOld": "cross-env PORT=10888 HTTPS=true react-scripts start",
     "start": "react-scripts start",
-    "startWin": "cross-env PORT=10888 HTTPS=true DANGEROUSLY_DISABLE_HOST_CHECK=true react-scripts start",
```
Comment (xyz1008) [inline package.json]: When add to git, Please enable this field.

File: src/hooks/useApi.ts [hunk 2]
```diff
@@ -10,9 +10,10 @@ const useApi = (route: string, options: any = {}) => {
     useEffect(() => {
         const fetchData = async () => {
             // @ts-ignore
-            const initData = window["Telegram"]['WebApp']['initData'];
-            // const initData = "query_id=AAGVFt08AwAAAJUW3TwV15BE&user=%7B%22id%22%3A7463573141%2C%22first_name%22%3A%22Abel%22%2C%22last_name%22%3A%22Tony%22%2C%22username%22%3A%22ProScriptSlinger4%22%2C%22language_code%22%3A%22en%22%2C%22allows_write_to_pm%22%3Atrue%7D&auth_date=1727671559&hash=2d32a0b1f98969c72bb9288ca8167f86605d5a2df2d4026af5166a21b8e4d19d";
+            // const initData = window["Telegram"]['WebApp']['initData'];
+            const initData = "query_id=AAGVFt08AwAAAJUW3TwV15BE&user=%7B%22id%22%3A7463573141%2C%22first_name%22%3A%22Abel%22%2C%22last_name%22%3A%22Tony%22%2C%22username%22%3A%22ProScriptSlinger4%22%2C%22language_code%22%3A%22en%22%2C%22allows_write_to_pm%22%3Atrue%7D&auth_date=1727671559&hash=2d32a0b1f98969c72bb9288ca8167f86605d5a2df2d4026af5166a21b8e4d19d";
             const url = getUrl() + route;
+            console.log(initData)
```
Comment (xyz1008) [inline src/hooks/useApi.ts]: Please remove console

File: src/hooks/useApi.ts [hunk 3]
```diff
@@ -10,9 +10,10 @@ const useApi = (route: string, options: any = {}) => {
     useEffect(() => {
         const fetchData = async () => {
             // @ts-ignore
-            const initData = window["Telegram"]['WebApp']['initData'];
-            // const initData = "query_id=AAGVFt08AwAAAJUW3TwV15BE&user=%7B%22id%22%3A7463573141%2C%22first_name%22%3A%22Abel%22%2C%22last_name%22%3A%22Tony%22%2C%22username%22%3A%22ProScriptSlinger4%22%2C%22language_code%22%3A%22en%22%2C%22allows_write_to_pm%22%3Atrue%7D&auth_date=1727671559&hash=2d32a0b1f98969c72bb9288ca8167f86605d5a2df2d4026af5166a21b8e4d19d";
```
Comment (xyz1008) [inline src/hooks/useApi.ts:14]: initData should be get from telegram

File: src/utils/api.ts [hunk 4]
```diff
@@ -36,8 +36,8 @@ export const fetchData = async (route: string, options: any = {}) => {
 
   /* const startTime = new Date().getTime(); */
   // @ts-ignore
-  const initData = window["Telegram"]["WebApp"]["initData"];
-  // const initData = "query_id=AAGVFt08AwAAAJUW3TwV15BE&user=%7B%22id%22%3A7463573141%2C%22first_name%22%3A%22Abel%22%2C%22last_name%22%3A%22Tony%22%2C%22username%22%3A%22ProScriptSlinger4%22%2C%22language_code%22%3A%22en%22%2C%22allows_write_to_pm%22%3Atrue%7D&auth_date=1727671559&hash=2d32a0b1f98969c72bb9288ca8167f86605d5a2df2d4026af5166a21b8e4d19d";
+  // const initData = window["Telegram"]["WebApp"]["initData"];
```
Comment (xyz1008) [inline src/utils/api.ts]: Here is same, have to get initData from telegram

**Review by xyz1008 — COMMENTED** [review]
Good, Please check this review and update

File: src/utils/api.ts (hunk 4 above)
Comment (louisbrant) [reply src/utils/api.ts]: ok , I will fix

**Pull request merged**